tests/integration/test_models/test_submodels/test_interface/test_butler_volmer.py

```diff
@@ -1,16 +1,15 @@
 #
 # Tests for the electrode-electrolyte interface equations
 #
-from tests import TestCase
+import pytest
 import pybamm
 from tests import get_discretisation_for_testing
 
-import unittest
 import numpy as np
 
 
-class TestButlerVolmer(TestCase):
-    def setUp(self):
+class TestButlerVolmer:
+    def setup_method(self):
```

Comment:
While it is fine here because we have both a setup method and a teardown method, I would suggest adding this item to the tracking issue as well – to migrate all of the `@pytest.fixture(autouse=True)` statements we've been using to the `pytest`-preferred style once all tests have been migrated. A single PR to do all of them would be great.

Reply:
I can cover this in one PR after all the test migration. I'll just have to find and replace it would be difficult to track all of them down now and add them to tracking issue.

Reply:
Makes sense. Done!